### ISSUE TITLE
Cannot find module 'twemoji.npm.js'

### ISSUE THREAD

**Issue by eyasuyuki** [marp-team/marp-cli#121]
I had an error when convert my Markdown to HTML.

```
$ npx @marp-team/marp-cli KotlinUpdates.md
Cannot find module '/Users/yasuyuki/.npm/_npx/40753/lib/node_modules/@marp-team/marp-cli/node_modules/twemoji/dist/twemoji.npm.js'. Please verify that the package.json has a valid "main" entry
```

My markdown is here: https://github.com/eyasuyuki/KotlinUpdates

**Comment by yhatt** [comment]
Now we cannot confirm because npm is downing by degraded state.
https://status.npmjs.org/incidents/0y69fmn96vp8

If you are in a hurry, you can use the standalone build that uses no dependent tools. (`marp-cli-v0.11.3-xxxxx.tar.gz`)
https://github.com/marp-team/marp-cli/releases

**Comment by yhatt** [comment]
There is a trouble in twemoji: https://github.com/marp-team/marp-cli/issues/121

**Comment by yhatt** [comment]
Fixed in twemoji v12.1.2. https://github.com/twitter/twemoji/issues/365#issuecomment-507775572
